client/service/src/config.rs

```diff
@@ -262,31 +262,43 @@ impl Default for RpcMethods {
 	}
 }
 
+#[static_init::dynamic(drop)]
```

Comment:
```suggestion
#[static_init::dynamic(drop,lazy)]
```

This probably doesn't need the extra performance that non-lazy initialization brings, and making it lazy will prevent an extra `.init_array` section from being generated and also prevent the initializer from having to run before `main` (which otherwise is a little silly since the only thing it's doing is just assigning `None` to it). Also, if something else would also run before `main` and would try to access this too early we wouldn't get a panic, so in theory it's a little safer.

Reply:
Uh, sorry, looks like `cargo fmt` doesn't like the lack of space after the `,`. (: